dmapiclient/data.py

```diff
@@ -563,15 +568,16 @@ def get_brief(self, brief_id):
         return self._get(
             "/briefs/{}".format(brief_id))
 
-    def find_briefs(self, user_id=None, status=None, framework=None, lot=None, page=None, human=None):
+    def find_briefs(self, user_id=None, status=None, framework=None, lot=None, page=None, human=None, with_users=None):
         return self._get(
             "/briefs",
             params={"user_id": user_id,
                     "framework": framework,
                     "lot": lot,
                     "status": status,
                     "page": page,
-                    "human": human
+                    "human": human,
+                    "with_users": with_users
```

Comment:
Turns out that if we ask for a `user_id`, [then it doesn't ever return briefs with users](https://github.com/alphagov/digitalmarketplace-api/blob/b8bb54b044daf663a9a6401a344fc0cc6c451e8c/app/main/views/briefs.py#L141).  

Nothing goes wrong, just that passing both parameters results in unexpected behaviour.

Maybe throw in a warning message like you did in the `find_users` method?